tests/devices/a1/message_a1_test.py

```diff
@@ -11,24 +11,41 @@
     MessageSet,
     NewProtocolTags,
 )
+from midealocal.message import BodyType, MessageType
 
 
 class TestMessageA1Base:
     """Test A1 Message Base."""
 
     def test_message_id_increment(self) -> None:
         """Test message Id Increment."""
-        msg = MessageA1Base(protocol_version=1, message_type=1, body_type=1)
-        msg2 = MessageA1Base(protocol_version=1, message_type=1, body_type=1)
+        msg = MessageA1Base(
+            protocol_version=1,
+            message_type=MessageType.test,
```

Comment:
Instead of creating a type test, maybe it is better to use an existing value.

Reply:
which one you prefer ?
It was "1" and was a non existing value, this is why I created a "test"

Reply:
Query will do the job.

Reply:
Done